### PR TITLE
Feature/#175 calendar feed 3

### LINKED ISSUE #175: As a student, I want to be able to subscribe to a calendar feed, which delivers up to date calender events for all lectures of all courses I am enrolled in (13).
*AC:*
- The calender-feed can be subscribed via url, which can be found in the account settings as it can be seen in the mock
- The calender feed updates according to changes the lecturer makes on his existing lectures in a course the student is enrolled in.
- The calender feed updates accordingly when new lectures are scheduled in a course the student is enrolled in.
- The calender feed - by default - does not add notifications to scheduled lectures.
- The calender feed titles the calender event with: Title Course - Title Lecture.

_Hint: 
https://www.figma.com/file/6yo4FXp2lLtE4uZvI0lsFr/Paper-Prototype?node-id=0%3A69 the students view contains the account view_

### PR DESCRIPTION
**Pull request by LucPrestin**
# Resolves #175

This PR changes / adds / introduces the feature of ....

Acceptance Criteria:

- [ ] The calender-feed can be subscribed via url, which can be found in the account settings as it can be seen in the mock
- [ ] The calender feed updates according to changes the lecturer makes on his existing lectures in a course the student is enrolled in.
- [ ]     The calender feed updates accordingly when new lectures are scheduled in a course the student is enrolled in.
- [ ]     The calender feed - by default - does not add notifications to scheduled lectures.
- [ ]     The calender feed titles the calender event with: Title Course - Title Lecture.

Definition Of Done:
- [x] [Class-Diagram](https://www.lucidchart.com/invitations/accept/705a122b-58a9-4dfa-8b16-c936bc0a46bc) updated

### CODE REVIEW

File: app/controllers/lectures_controller.rb [hunk 1]
```diff
@@ -21,6 +21,7 @@ def index
   # GET courses/:course_id/lectures/1
   def show
     @current_user = current_user
+    @feedback = Feedback.new
```
Comment (m0e33) [inline app/controllers/lectures_controller.rb]: What is this doing here?

File: spec/models/user_spec.rb [hunk 2]
```diff
@@ -23,6 +23,12 @@
     expect(@user).not_to be_valid
   end
 
+  it "(student) is not valid without a calender url" do
```
Comment (m0e33) [inline spec/models/user_spec.rb]: Why are there brackets?

Comment (phoeinx) [reply spec/models/user_spec.rb]: I dunno, that's Lucs code, but I guess we could just leave them out.

File: app/controllers/lectures_controller.rb (hunk 1 above)
Comment (phoeinx) [reply app/controllers/lectures_controller.rb]: Dunno, there seems to be some weird stuff going on with feedback anyhow - looking at the feedback controller e.g.

Comment (m0e33) [reply app/controllers/lectures_controller.rb]: in the feedback controller the weird stuff going on is authored by you according to git.

Comment (phoeinx) [reply app/controllers/lectures_controller.rb]: Jup, found the issues - this branch is not based on the last version of the remote dev branch but on a history of merges of the remote dev branch with my local dev branch. Therefore some weird bits of code were dragged on. Looking into how to solve that as clean as possible now. Ideas welcome :D

Comment (ma-ro) [reply app/controllers/lectures_controller.rb]: Just delete the weird feedback code lines. I think its just the lectures_controller and the feedbacks controller.

File: app/controllers/feedbacks_controller.rb [hunk 3]
```diff
@@ -12,6 +12,17 @@ def create
       @feedback = @lecture.feedbacks.where(user_id: current_user.id)
       @feedback.update(comment_params)
     end
+
+    respond_to do |format|
```
Comment (ma-ro) [inline app/controllers/feedbacks_controller.rb]: I guess this needs to be deleted too

File: app/controllers/lectures_controller.rb (hunk 1 above)
Comment (phoeinx) [reply app/controllers/lectures_controller.rb]: yep did just that as in deleted my local dev branch, pulled the remote dev branch and compared the actual dev branch to our feature branch. Pushed commit and should be fine now

**Pull request merged**